src/group_theory/free_group.lean

```diff
@@ -19,7 +19,7 @@ functor from groups to types, see `algebra/category/Group/adjunctions`.
   modulo the relation `a * x * x⁻¹ * b = a * b`.
 * `mk`: the canonical quotient map `list (α × bool) → free_group α`.
 * `of`: the canoical injection `α → free_group α`.
-* `to_group f`: the canonical group homomorphism `free_group α →* G` given a group `G` and a
+* `lift f`: the canonical group homomorphism `free_group α →* G` given a group `G` and a
```

Comment:
lift should have similar docstrings throughout.

Reply:
Any suggestions for how best to document it?